hyperledger-fabric/fabric-tools/README.md

```diff
@@ -0,0 +1,89 @@
+# fabric-tools
```

Comment:
We should think about moving this cli to dovetail-cli. It will make easier the integration later when we want to deploy from the UI